examples/example-revenue-report/src/templates/result.php

```diff
@@ -0,0 +1,83 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="x-ua-compatible" content="ie=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Revenue report</title>
+</head>
+<body>
+<h1>Revenue report</h1>
+<form>
+    <table>
+        <tr>
+            <th>Service Plan</th>
+            <th>Total invoiced</th>
+            <th>Total paid</th>
+        </tr>
+        <?php foreach ($servicePlans as $servicePlan) { ?>
+            <tr>
+                <td><?php echo htmlspecialchars($servicePlan['name'], ENT_QUOTES); ?></td>
+                <td><?php echo number_format($servicePlan['totalIssued'], 2) . ' ' . $currency; ?></td>
+                <td><?php echo number_format($servicePlan['totalPaid'], 2) . ' ' . $currency; ?></td>
+            </tr>
+        <?php } ?>
+    </table>
+</form>
+<div id="chart-total-invoiced" style="width: 500px; height: 350px;"></div>
```

Comment:
reformat the HTML to use indentation

Reply:
This is how PHPStorm formatted it. It's not bad imo.